src/Skoruba.IdentityServer4/Views/Account/Login.cshtml

```diff
@@ -17,7 +17,7 @@
 						<form asp-route="Login">
 							<input type="hidden" asp-for="ReturnUrl" />
 
-							<div class="alert alert-info" role="alert">Use username - <strong>bob</strong>, password - <strong>password</strong></div>
+							<div class="alert alert-info" role="alert">Use username - <strong>bob</strong>, password - <strong>Pa$$word123</strong></div>
```

Comment:
Could this be pulled dynamically from the seed data? Just so that you don't have to update it in multiple places.